transform/mattermost-analytics/models/intermediate/product/nps/int_nps_feedback.sql

```diff
@@ -0,0 +1,33 @@
+with mattermost_nps as (
+select 
+    distinct server_id as server_id
```

Comment:
Is the distinct really needed?

Reply:
Removed distinct and updated to qualify. distinct is bringing lot of duplicate rows due to `timestamp` field